include/stylesheet.css

```diff
@@ -260,6 +261,40 @@ img.circle {
     cursor: pointer;
 }
 
+/* tab switcher tooltips */
+.circle:hover .tab_tooltip {
+    position:absolute;
+    top:22px;
```

Comment:
Having the tooltip appear above the button is more visible and won't be obscured by the mouse cursor, -35px for top looks good for this